packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DataAccessRequestAccessorsFilesForm.tsx

```diff
@@ -80,58 +126,24 @@ export default function DataAccessRequestAccessorsFilesForm(
     researchProjectId,
     onCancel,
   } = props
-  const { data: user } = useGetCurrentUserProfile()
+  const { accessToken } = useSynapseContext()
+  const isLoggedIn = Boolean(accessToken)
+  const { data: user } = useGetCurrentUserProfile({ enabled: isLoggedIn })
   const [alert, setAlert] = useState<AlertProps | undefined>()
-  const [dataAccessRequest, setDataAccessRequest] = useState<
-    Request | Renewal | undefined
+  const [accessorChanges, setAccessorChanges] = useState<
+    AccessorChange[] | undefined
   >()
+  const [summaryOfUse, setSummaryOfUse] = useState<string | undefined>()
+  const [publication, setPublication] = useState<string | undefined>()
```

Comment:
Store non-attachment fields in local state. attachments will be immediately synced with the server.